roles/ovirt-repositories/tasks/rh-subscription.yml

```diff
@@ -0,0 +1,8 @@
+---
+- name: Register and subscribe to multiple pools.
+  redhat_subscription:
+    state: present
+    username: "{{ rh_username }}"
```

Comment:
I think it would be better to use rh_username|mandatory , rather then set the default value to `""`